### PR TITLE
chore: add catalog-info.yml

### PR DESCRIPTION
**Pull request by ichintanjoshi**
## Description

- Add catalog-info.yaml

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ichintanjoshi! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-components-tinymce-advanced-plugins'
+  description: "the standard edX header for use in React applications"
```
Comment (feanil) [inline catalog-info.yaml:8]: This description seems wrong, it looks like the README has a typo.  I think you can say:

"A set of tinymce plugins to use in react applications."

**Comment by mphilbrick211** [comment]
Flagging this, @ichintanjoshi!

**Comment by ichintanjoshi** [comment]
@mphilbrick211  Thanks for reminder, it's updated. All checks are green now @feanil

**Review by feanil — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@ichintanjoshi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-components-tinymce-advanced-plugins&entry.867055334=https://github.com/openedx/frontend-components-tinymce-advanced-plugins/pull/16&entry.1484655318=https://github.com/ichintanjoshi&entry.752974735=2024-05-30+06:01&entry.1917517419=2024-09-03+16:21&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**